src/net/tls.cc

```diff
@@ -46,6 +47,8 @@
 
 namespace seastar {
 
+logger tlslog("tls");
```

Comment:
It can be defined static actually.
And maybe it should be called seastar_tlslog, to make its scope clearer.

Reply:
Should TLS have a log? Why? We don't have loggers for TCP etc? Should not any logger be user-provided for this level of library function?

Reply:
Maybe we should have finer granularity loggers so their level can be controlled in tests or production.
But this can be discussed orthogonally.
Currently my main concern is getting the fix in.